src/test/java/io/fabric8/tests/integration/KubernetesIntegrationKT.java

```diff
@@ -16,25 +16,38 @@
 
 package io.fabric8.tests.integration;
 
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.api.model.PodList;
+import io.fabric8.kubernetes.api.model.PodCondition;
+import io.fabric8.kubernetes.client.internal.readiness.Readiness;
 import io.fabric8.kubernetes.client.KubernetesClient;
 
+import org.arquillian.cube.kubernetes.api.Session;
+
 import org.jboss.arquillian.container.test.api.RunAsClient;
 import org.jboss.arquillian.junit.Arquillian;
 import org.jboss.arquillian.test.api.ArquillianResource;
+import static org.junit.Assert.assertTrue;
```

Comment:
Probably we should organise imports before committing. Better to separate `import static` from plain `import`s.